src/SubMenu/styledComponents.tsx

```diff
@@ -56,6 +56,7 @@ const level1Style = ({ selected, theme }: StyleCalculationInput) => css`
   ${defaultLabelStyle({ theme, selected })}
   background: ${theme.colors.white};
   animation: all 1s;
+  width: 100%;
```

Comment:
submenus in File Repository facet panel weren't filling up the width of the column